hax/treemakers/common.py

```diff
@@ -126,6 +130,22 @@ def extract_data(self, event):
                 # Alternative S1 interaction
                 result['alt_s1_interaction_z'] = it.z
 
+        result['largest_other_s2_delay_s1'] = float('nan')
+        result['largest_other_s2_delay_s2'] = float('nan')
+        result['largest_other_s2_pattern_fit'] = float('nan')
+
+        for ix in event.s2s:
+            if (ix != interaction.s2) and (event.peaks[ix].hit_time_mean - s1.hit_time_mean > 0):
+                # largest other s2 after main s1
+                pk = event.peaks[ix]
+                result['largest_other_s2_delay_s1'] = pk.hit_time_mean - s1.hit_time_mean
```

Comment:
It looks like this is not necessary since we have a branch called alt_s2_interaction_z.